app/models/work.rb

```diff
@@ -1,4 +1,4 @@
-# typed: strict
+# typed: false
```

Comment:
Why?

Reply:
srb did it for me to be helpful, apparently, but I manually reverted this and srb is still happy. thanks!